app/boards/arm/waveshare_rp2040_zero/waveshare_rp2040_zero_defconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023 The ZMK Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_RP2XXX=y
+CONFIG_SOC_RP2040=y
+CONFIG_BOARD_WAVESHARE_RP2040_ZERO=y
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=240000000
```

Comment:
```suggestion
CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=125000000
```

I had to fix this in our other boards recently, they had the wrong sys clock frequency set.